main:
  hz                    : 50 
  spinner_count         : 1

alive:
  hz                   : 20          ## heartbeat 보고 주기

motor:
  wheel_base           : 50         ## wheel base
  track_width          : 50         ## track width

FrontLeft:
  hz          : 50                    ## motor thread 동작 속도 
  comm_type   : "Modbus485"           ## 통신 방식 (ModbusTCP, RTU(232, 485))
  device      : "/dev/tty_WHEEL_FL"   ## device name
  port        : 0                     ## port number
  baud        : 115200                ## baudrate
  slave_id    : 1                     ## slave id
  retry_count : 3                     ## 통신 연결 시도 횟수
  retry_turm  : 2                     ## 통신 연결 시도 간격
  debug       : 1                     ## debug 옵션 (0: off, 그외 : on)

  coefficient : 1.0                   ## traction motor speed coefficient
  traction_id : 1                     ## traction motor id
  steering_id : 2                     ## steering motor id

  home_position : 63000               # mechanical origin
  left_limit    : 4048000             ## positive_sensor_limit - 12000(homing mode default offset value)
  right_limit   : -419800             ## negative_sensor_limit + 12000(homing mode default offset value)

FrontRight:
  hz          : 50                    ## motor thread 동작 속도 
  comm_type   : "Modbus485"           ## 통신 방식 (ModbusTCP, RTU(232, 485))
  device      : "/dev/tty_WHEEL_FR"   ## device name
  port        : 0                     ## port number
  baud        : 115200                ## baudrate
  slave_id    : 1                     ## slave id
  retry_count : 3                     ## 통신 연결 시도 횟수
  retry_turm  : 2                     ## 통신 연결 시도 간격
  debug       : 1                     ## debug 옵션 (0: off, 그외 : on)

  coefficient : 1.0                   ## traction motor speed coefficient
  traction_id : 1                     ## traction motor id
  steering_id : 2                     ## steering motor id

  home_position : 63000               # mechanical origin
  left_limit    : 4048000             ## positive_sensor_limit - 12000(homing mode default offset value)
  right_limit   : -419800             ## negative_sensor_limit + 12000(homing mode default offset value)  

RearLeft:
  hz          : 50                    ## motor thread 동작 속도 
  comm_type   : "Modbus485"           ## 통신 방식 (ModbusTCP, RTU(232, 485))
  device      : "/dev/tty_WHEEL_RL"   ## device name
  port        : 0                     ## port number
  baud        : 115200                ## baudrate
  slave_id    : 1                     ## slave id
  retry_count : 3                     ## 통신 연결 시도 횟수
  retry_turm  : 2                     ## 통신 연결 시도 간격
  debug       : 1                     ## debug 옵션 (0: off, 그외 : on)

  coefficient : -1.0                  ## traction motor speed coefficient
  traction_id : 1                     ## traction motor id
  steering_id : 2                     ## steering motor id

  home_position : 63000               # mechanical origin
  left_limit    : 4048000             ## positive_sensor_limit - 12000(homing mode default offset value)
  right_limit   : -419800             ## negative_sensor_limit + 12000(homing mode default offset value)  

RearRight:
  hz          : 50                    ## motor thread 동작 속도 
  comm_type   : "Modbus485"           ## 통신 방식 (ModbusTCP, RTU(232, 485))
  device      : "/dev/tty_WHEEL_RR"   ## device name
  port        : 0                     ## port number
  baud        : 115200                ## baudrate
  slave_id    : 1                     ## slave id
  retry_count : 3                     ## 통신 연결 시도 횟수
  retry_turm  : 2                     ## 통신 연결 시도 간격
  debug       : 1                     ## debug 옵션 (0: off, 그외 : on)

  coefficient : -1.0                  ## traction motor speed coefficient
  traction_id : 1                     ## traction motor id
  steering_id : 2                     ## steering motor id

  home_position : 63000               # mechanical origin
  left_limit    : 4048000             ## positive_sensor_limit - 12000(homing mode default offset value)
  right_limit   : -419800             ## negative_sensor_limit + 12000(homing mode default offset value)

logger:
  debug               : 1             ## debug 옵션 (0: off, 그외 : on)
  hz                  : 50            ## thread 동작 속도
  suffix              : _driving      ## 로그 파일 이름 뒤에 붙는 suffix
  dir_name            : log           ## 로그 파일 저장 디렉토리
  max_log_per_write   : 50            ## 한번에 기록 할 최대 로그 갯수
  auto_write_interval : 500           ## millisecond 로그 파일을 자동으로 기록하는 주기